src/commands/swap.ts

```diff
@@ -0,0 +1,178 @@
+import { flags } from "@oclif/command";
+import { Interfaces } from "@uns/ark-crypto";
+import { Network } from "@uns/ts-sdk";
+import { CryptoAccountPassphrases } from "types";
+import { cli } from "cli-ux";
+
+import { BaseCommand } from "../baseCommand";
+import { WriteCommand } from "../writeCommand";
+import { SendCommandHelper } from "../commandHelpers/send-helper";
+import { Formater, NestedCommandOutput, OUTPUT_FORMAT } from "../formater";
+import {
+    checkFlag,
+    getWalletAddress,
+    getWalletFromPassphrase,
+    isDid,
+    isTokenId,
+    fromSatoshi,
+    toSatoshi,
+} from "../utils";
+
+const ethAddressRegex = /^0x[a-fA-F0-9]{40}$/;
+
+// TODO: Include real ETH transaction fees in the swap (amount of wUNS tokens to not mint, as a fee)
+/**
+ * Keep 100 uns as fee now for testing
+ *
+ * This is just for showing a preview message as it is the bridge-core that will do the wUNS minting and substract the fee!
+ */
+const getSwapCost = async () => 100_00000000;
+
+// FIXME: Put the bridge wallet addresses in the UNS SDK
+const getBridgeWalletAddress = (network: Network) => {
+    switch (network) {
+        case "livenet":
+            throw new Error("Livenet swap is not supported yet");
+        case "sandbox":
+            return "Sc68Hu6hRTCy9z4b7ppoeD24AX227fZ5UX";
+        // @ts-ignore
+        case "dalinet":
+            if (process.env.NODE_ENV === "test") return "Sc68Hu6hRTCy9z4b7ppoeD24AX227fZ5UX";
+        default:
+            throw new Error("Unsupported swap network");
+    }
+};
```

Comment:
Where to store this information is a real design point to consider.
If this cryptoaccount address must be changed, this will be very hard if is hard-coded many CLI installations, not updated.

My proposal: using a property of a specific @unikname, with the same explicit value on all networks.
Same code for all environments, possibility to update the cryptoaccount address.

What do you think?

Reply:
> If this cryptoaccount address must be changed, this will be very hard if is hard-coded in many CLI installations, not up-to-date.

Good point. 

> My proposal: using a property of a specific @unikname, with the same explicit value on all networks.

Good idea, its passphrase should be as private as the exchange's one then.